app/services/send_candidate_rejection_email.rb

```diff
@@ -6,16 +6,20 @@ def initialize(application_choice:)
   end
 
   def call
-    candidate_application_choices = application_choice.self_and_siblings
-    number_of_pending_decisions = candidate_application_choices.awaiting_provider_decision.count
-    number_of_offers = candidate_application_choices.offer.count
+    if FeatureFlag.active?(:structured_reasons_for_rejection)
+      SendRejectionEmailToCandidate.new(application_choice: application_choice).call
```

Comment:
I wonder about breaking out into a new service to call back to the CandidateMailer, not a blocker, more of an opinion but it seems like we could reduce a bit of code if we had one method that could return the `CandidateMailer` method needed. The resulting call is always `CandidateMailer.send(:mailer_method, application_choice).deliver_later`

Reply:
I considered this as a transitional step until the flag is removed and structured reasons for rejection takes over. Then the existing code can be canned. However, I now wonder if the flag is completely unecessary and instead I should be checking and including the correct reasons (structured or single reason as decline by default is already handled).